client/dashboard/customizable.js

```diff
@@ -226,11 +242,32 @@ class CustomizableDashboard extends Component {
 }
 
 export default compose(
-	withSelect( select => {
-		const { getCurrentUserData } = select( 'wc-api' );
+	withSelect( ( select, props ) => {
+		const { getCurrentUserData, getProfileItems, getOptions } = select( 'wc-api' );
 		const userData = getCurrentUserData();
 
+		const profileItems = getProfileItems();
+		const taskListHidden =
+			'yes' ===
+			get(
+				getOptions( [ 'woocommerce_task_list_hidden' ] ),
+				[ 'woocommerce_task_list_hidden' ],
+				'no'
+			);
+
+		const tasks = getTasks( {
+			profileItems,
+			options: getOptions( [ 'woocommerce_onboarding_payments' ] ),
+			query: props.query,
+		} );
+
+		const visibleTasks = filter( tasks, task => task.visible );
+		const completedTasks = filter( tasks, task => task.completed );
```

Comment:
I think this logic may need a little tweaking.  It's possible to have a completed task that is not visible which would potentially create a completed list when there are incomplete items.

<img width="484" alt="Screen Shot 2019-10-10 at 10 25 08 AM" src="https://user-images.githubusercontent.com/10561050/66534372-93b81c00-eb48-11e9-83a9-3c95412288af.png">

Maybe compare based on completed and visible?

```suggestion
		const completedTasks = filter( tasks, task => task.completed && task.visible );
```